src/validation_workflow/validation_args.py

```diff
@@ -0,0 +1,51 @@
+# Copyright OpenSearch Contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+import argparse
+
+from .rpm import Rpm
+from .tar import Tar
+from .yum import Yum
```

Comment:
Instead of writing relative path it is better to write the full path of these libs.
Also it is better to give a name with full contaxt of these classes.

Example: `ValidationRpm`, `ValidationTar`.